lisa/tools/kernel_config.py

```diff
@@ -22,20 +23,27 @@ def can_install(self) -> bool:
         return False
 
     def is_built_in(self, config_name: str) -> bool:
-        return (
-            self.node.execute(
-                f"grep ^{config_name}=y {self.config_path}", sudo=True, shell=True
-            ).exit_code
-            == 0
-        )
+        if isinstance(self.node.os, BSD):
```

Comment:
It looks very different on each method, it's worth to create a subclass for BSD.